packages/dashboard/src/components/app-base.tsx

```diff
@@ -1,16 +1,7 @@
-import {
-  Alert,
-  AlertProps,
-  createTheme,
-  CssBaseline,
-  GlobalStyles,
-  Grid,
-  Snackbar,
-} from '@mui/material';
+import { Alert, AlertProps, createTheme, CssBaseline, Grid, Snackbar } from '@mui/material';
 import { ThemeProvider } from '@mui/material/styles';
 import React from 'react';
-import { rmfDark, rmfDarkLeaflet, rmfLight } from 'react-components';
```

Comment:
Remove the themes in `react-components` as well.

I know the themes have many issues both design and implementation wise, but dark mode was one of the requested feature as well.

Reply:
Gotcha, it was probably introduced before I was aware of it. I'll see if I can just retain it and work around the new Theme props and API